python使用被称为异常的特殊对象来管理程序，
异常使用try-except模块处理的
